infoschema/infoschema_test.go

```diff
@@ -120,7 +119,7 @@ func (*testSuite) TestT(c *C) {
 
 	schemaNames := is.AllSchemaNames()
 	c.Assert(schemaNames, HasLen, 3)
-	c.Assert(testutil.CompareUnorderedStringSlice(schemaNames, []string{infoschema.Name, perfschema.Name, "Test"}), IsTrue)
+	c.Assert(testutil.CompareUnorderedStringSlice(schemaNames, []string{infoschema.Name, "PERFORMANCE_SCHEMA", "Test"}), IsTrue)
```

Comment:
`perfschema.Name` is still in `perfschema/const.go`?

Reply:
perfschema imports infoschema ... (call `infoschema.RegisteVirtualTable()`)
If infoschema imports perfschema here, there will be a circle dependancy
@lysu